content/vi/docs/reference/glossary/cri-o.md

```diff
@@ -0,0 +1,27 @@
+---
+title: CRI-O
+id: cri-o
+date: 2020-03-05
+full_link: https://cri-o.io/#what-is-cri-o
+short_description: >
+  A lightweight container runtime specifically for Kubernetes
+  Một lightweight container runtime đặc biệt cho Kubernetes

```

Comment:
The English is still here.

Reply:
I would expect to see “lightweight” translated.

Reply:
thank you! I will update it